test/unit_test/deepdoc/parser/test_monkeyocrv2_parser.py

```
@@ -0,0 +1,36 @@
+        content = make_zip("doc", "hello")
+        def raise_for_status(self): pass
+    monkeypatch.setattr("deepdoc.parser.monkeyocrv2_parser.requests.post", lambda *a, **k: Response())
+    sections, tables = MonkeyOCRv2Parser("http://parser").parse_pdf("doc.pdf", binary=b"pdf", page_from=1, page_to=3)
+    assert sections == [("hello", "@@2\t1\t30\t2\t40##")]
+    assert tables == []
+
+
+def test_sends_page_range_and_accepts_binary(monkeypatch):
+    seen = {}
+    class Response:
+        headers = {"content-type": "application/zip"}; content = make_zip("doc", "ok")
```

Ruff (0.16.2)

[warning] Mutable default value for class attribute

(RUF012)